### PR TITLE
[GRAPH-1110] Merge in v1.7.7

### PR DESCRIPTION
**Pull request by bryanjos**
Merges in the 1.7.7 tag from Absinthe
https://github.com/absinthe-graphql/absinthe/releases/tag/v1.7.7

### CODE REVIEW

File: test/absinthe/execution/subscription_test.exs [hunk 1]
```diff
@@ -452,6 +452,31 @@ defmodule Absinthe.Execution.SubscriptionTest do
...
+               @query,
+               Schema,
+               variables: %{"clientId" => "abc"},
+               context: %{pubsub: PubSub, authorized: false}
+             )
+
+    assert_received {:telemetry_event, {[:absinthe, :execute, :operation, :start], _, _, _}}
+
+    assert_received {:telemetry_event, {[:absinthe, :execute, :operation, :stop], _, _, _}}
+  end
```
Comment (michaelcaterisano) [inline test/absinthe/execution/subscription_test.exs:478]: Oh right, I added this :) Now we can remove the similar test in massdriver

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,17 @@
 # Changelog
 
-## Unreleased
+## 1.7.7
+- POTENTIALLY BREAKING Bug Fix: [Validate variable usage in nested input arguments](https://github.com/absinthe-graphql/absinthe/pull/1290).This could break incoming documents previously considered valid. Skip the Absinthe.Phase.Document.Arguments.VariableTypesMatch phase to avoid this check. See Absinthe.Pipeline on adjusting the document pipeline.
+- #1321 resolves telemetry issues
+- Various minor dependency versioning tweaks
+- Handle Elixir 1.17 warnings
+
```
Comment (michaelcaterisano) [inline CHANGELOG.md:8]: I totally forgot to add my changes here. Maybe we should ticket updating the Absinthe docs at some point, they're missing a lot of stuff.

**Review by michaelcaterisano — APPROVED** [review]
ty!

File: CHANGELOG.md (hunk 2 above)
Comment (bryanjos) [reply CHANGELOG.md:8]: Yeah, any doc updates there would be beneficial

**Pull request merged**